engine-tests/src/test/java/org/terasology/documentation/ModuleMarkupScraper.java

```diff
@@ -48,69 +50,85 @@
 
 public final class ModuleMarkupScraper {
 
-    private static final DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
     private static final String DEFAULT_GITHUB_MODULE_URL = "https://github.com/Terasology/";
     private static final List INTERNAL_MODULES = Arrays.asList("Core", "engine", "CoreSampleGameplay", "BuilderSampleGameplay", "BiomesAPI");
-    private static final Comparator<? super Module> moduleInfoComparator = Comparator.comparing(o -> o.getMetadata()
+    private static final Comparator<? super Module> MODULE_INFO_COMPARATOR  = Comparator.comparing(o -> o.getMetadata()
             .getDisplayName().toString());
 
-    protected static Context context;
-
-    private static ModuleManager moduleManager;
-
-    private static HeadlessEnvironment env;
-
-    private ModuleMarkupScraper() throws IOException {
-        super();
-    }
-
     /**
      * @param args (ignored)
      * @throws Exception if the module environment cannot be loaded
      */
     public static void main(String[] args) throws Exception {
-        env = new HEnv(new Name("engine"));
-        context = env.getContext();
-        moduleManager = context.get(ModuleManager.class);
+         HeadlessEnvironment env = new HEnv(new Name("engine"));
+         Context context = env.getContext();
+         ModuleManager moduleManager = context.get(ModuleManager.class);
+
+//       ModuleManager moduleManager = ModuleManagerFactory.create("meta.terasology.org");
```

Comment:
Why is this commented? Is using the module manager provided by `context.get(ModuleManager.class)` sufficient?

Reply:
`ModuleManagerFactory.create(masterServerAddress)` occurs within `ModuleMetaDataProvidingHeadlessEnvironment.setupModuleManager()`. So at this point `context.get()` pulls the created `ModuleManager `.

However, there were other redundant game objects being loaded within `ModuleMetaDataProvidingHeadlessEnvironment` that I was able to remove.

I'm loading assets and anything related to reflection. I avoid